src/components/dashboard/SendLinkSummary.js

```diff
@@ -174,7 +174,7 @@ const SendLinkSummary = ({ screenProps, styles }: AmountProps) => {
             })
           }
 
-          fireEvent('SEND_DONE', { type: screenState.params.type || 'Address' })
```

Comment:
definitely not. what makes you think you can simply remove it?
if it is a null issue then do get(screenState,'params.type')

Reply:
I wasn't sure of it that is why it is put up for a review, to get your opinion about it

Reply:
btw why was it put in there? it was placed there a few days ago and produced this bug

Reply:
because it was a bug, not logging the correct type of the send.
we have send by qr, send from payment request, send by payment and currently inactive send to address.